src/Symfony/Component/Console/Application.php

```diff
@@ -1067,6 +1070,11 @@ public function setDefaultCommand($commandName)
         $this->defaultCommand = $commandName;
     }
 
+    /**
+     * @param string $string
+     * 
+     * @return int
+     */
```

Comment:
Symfony version (count is `src` folder only)
3.0 (a84ea7ab70356662be48491f36e30cbfcb2d7302): 
Number of private methods: 1005, with PHPDoc 589, with comment 0
2.5 (5c83d12f2fbe93622e6ae2aad72f4033adbebe36):
Number of private methods: 873, with PHPDoc 546, with comment 0




(but maybe my script is off?)